### PR TITLE
Dynamic portfolio section

### PR DESCRIPTION
**Pull request by this-ezzy**
I added the html, css and javascript logic for the dynamic portfolio section, to enable the information in this section be generated dynamically. This is a pull request to enable me merge this branch into the main branch.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -315,6 +331,118 @@ menu {
...
+  padding: 16px;
+  width: 100%;
+  height: 100%;
+  align-items: flex-start;
+}
+
+.modal_img_container {
+  position: relative;
+  width: 100%;
+}
```
Comment (jerryowusu) [inline styles.css]: Kindly ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins). 
For the mobile version, kindly ensure that the modal covers the whole page, also the image and the close button should be just like in the figma design.

### Your mobile design

Screenshot from 2022-12-21 07-10-24

### Your desktop design 

Screenshot from 2022-12-21 07-10-52


### Figma mobile design


Screenshot from 2022-12-21 07-23-39


### Figma desktop design


Screenshot from 2022-12-21 07-24-11

File: styles.css [hunk 2]
```diff
@@ -315,6 +331,118 @@ menu {
...
+  padding: 0 10px;
+  font-size: 1.5rem;
+  color: #fff;
+  background: #67798e;
+  width: 2rem;
+  position: absolute;
+  top: 30px;
+  right: 25px;
+  z-index: 1000;
+}
```
Comment (jerryowusu) [inline styles.css]: Kindly ensure that your close button has the same background and position as the figma design for both mobile and desktop to match the design. :wink:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi Team 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

so-close


## Highlights
All linters are passing :heavy_check_mark: 
Created dynamic modal using javascript :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 3]
```diff
@@ -656,10 +791,141 @@ form {
...
+  .modal {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    display: none;
+    overflow: scroll;
+    justify-content: center;
+    background: #c1c7d0;
+  }
```
Comment (Owusu-Desmond) [inline styles.css]: - [ ] As requested by the previous code reviewer, From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
  - On the desktop, your modals should have margin-top and margin-bottom to match the figma design

| Your Design |Figma Design |
| :---:   | :---: | 
| Screenshot from 2022-12-21 09-04-29| image|

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Ezzywealth  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @Ezzywealth , 

I am Favour 👩🏽 , your reviewer for this PR.

Great Job implementing all the required changes suggested by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**